cvat/apps/engine/task.py

```diff
@@ -252,25 +253,33 @@ def update_progress(progress):
     compressed_chunk_writer = compressed_chunk_writer_class(db_data.image_quality)
     original_chunk_writer = original_chunk_writer_class(100)
 
+    # calculate chunk size if it isn't specified
+    if db_data.chunk_size is None:
```

Comment:
@azhavoro , will it be critical for current video decoder if somebody specified a wrong chunk_size (too small or even)?